docs/demo.mdx

```diff
@@ -0,0 +1,372 @@
+---
+title: Internal Demo Page for Styling
+---
+
+import '@site/node_modules/infima/dist/css/default/default.css';
```

Comment:
```suggestion
```

Do we need it?

Reply:
No. It was also something I thought was needed from the beginning but doesn't have any purpose, in the end. Thanks!